lib/backend/k8s/conversion/conversion_test.go

```diff
@@ -227,7 +213,7 @@ var _ = Describe("Test Pod conversion", func() {
 		Expect(wep.Value.(*apiv2.WorkloadEndpoint).Spec.Endpoint).To(Equal("eth0"))
 		Expect(wep.Value.(*apiv2.WorkloadEndpoint).Spec.Orchestrator).To(Equal("k8s"))
 		Expect(len(wep.Value.(*apiv2.WorkloadEndpoint).Spec.IPNetworks)).To(Equal(1))
-		Expect(wep.Value.(*apiv2.WorkloadEndpoint).ObjectMeta.Labels).To(Equal(map[string]string{"calico/k8s_ns": "default"}))
+		Expect(wep.Value.(*apiv2.WorkloadEndpoint).ObjectMeta.Labels).To(BeNil())
```

Comment:
Don't think this test is right - should have the orchestrator / namespace labels yeah?

Reply:
well that is sad.  Let me investigate.